packages/core/src/app/customer/CheckoutButtonContainer.tsx

```diff
@@ -54,8 +56,16 @@ const CheckoutButtonContainer: FunctionComponent<CheckoutButtonContainerProps &
         return null;
     }
 
+    const isPaypalCommerce = methodIds.includes('paypalcommerce');
+
+    if (isPaypalCommerce && isPaymentStepActive) {
+        return null;
+    }
+
     return (
-        <div className='checkout-button-container'>
+        <div className='checkout-button-container'
+             style={ isPaymentStepActive ? { position: 'absolute', left: '100%', top: '-100%' } : undefined }
```

Comment:
Is it possible to use { position: 'absolute', visibility: 'hidden' } in this case? Because we might have some issues in mobile Safari, when the user quickly scrolling up and down.

Reply:
My concern is changing `visibility` may cause some buttons to react. 
I did notice an issue on mobile devices, white space on the right side of the page when the buttons are hidden, hence I set the left to `0` so that the page size should stay the same.